app/javascripts/renderer/renderer.js

```diff
@@ -137,48 +130,49 @@ async function registerIpcMessageListener() {
     const message = payload.message;
     const data = payload.data;
 
-    if(message === "window-blurred") {
+    if (message === "window-blurred") {
```

Comment:
We should constantize/enum these messages. i.e:

const IpcMessages = {
   WindowBlurred: 'window-blurred',
    ...
}